code/modules/mob/living/basic/farm_animals/bee/_bee.dm

```diff
@@ -130,15 +129,6 @@
 	if(is_queen)
 		beehome.queen_bee = src
 
-/mob/living/basic/bee/proc/apply_chemicals(mob/living/basic/attacker, atom/target)
```

Comment:
why was this removed, is it done elsewhere?

Reply:
yes. he get a venemous element in assign_reagent() so this is dont need now